client/core/core.go

```diff
@@ -1542,13 +1542,34 @@ func (c *Core) Logout() error {
 // initializeDEXConnections connects to the DEX servers in the conns map and
 // authenticates the connection. If registration is incomplete, reFee is run and
 // the connection will be authenticated once the `notifyfee` request is sent.
+// If an account is not found on the dex server upon dex authentication the
+// account is disabled and the corresponding entry in c.conns is removed
+// which will result in the user being prompted to register again.
 func (c *Core) initializeDEXConnections(crypter encrypt.Crypter) []*DEXBrief {
 	// Connections will be attempted in parallel, so we'll need to protect the
 	// errorSet.
 	var wg sync.WaitGroup
 	c.connMtx.RLock()
-	defer c.connMtx.RUnlock()
 	results := make([]*DEXBrief, 0, len(c.conns))
+	var reconcileConnectionsWg sync.WaitGroup
+	reconcileConnectionsWg.Add(1)
+	disabledAccountHostChan := make(chan string)
+	var disabledAccountHosts []string
+	// If an account has been disabled the entry is removed from c.conns.
+	go func() {
+		defer reconcileConnectionsWg.Done()
+		for disabledAccountHost := range disabledAccountHostChan {
+			disabledAccountHosts = append(disabledAccountHosts, disabledAccountHost)
+		}
+		if len(disabledAccountHosts) > 0 {
+			c.connMtx.Lock()
+			for _, disabledAccountHost := range disabledAccountHosts {
```

Comment:
We need a log at at least Warning level that informs the user of what happened.

Reply:
Done.